dashboards/kubernetes-prometheus-kubelet/kubernetes-prometheus-kubelet.json

```diff
@@ -0,0 +1,799 @@
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT latest(process_resident_memory_bytes) FROM Metric WHERE ((cluster_name={{cluster}}) AND (job = 'kubelet')) SINCE 60 MINUTES AGO UNTIL NOW FACET cluster_name LIMIT max TIMESERIES 30000"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "yAxisLeft": {
+              "zero": false
+            }
+          }
+        },
+        {
+          "title": "CPU usage ",
+          "layout": {
+            "column": 5,
+            "row": 37,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT rate(sum(process_cpu_seconds_total), 1 SECONDS) FROM Metric WHERE ((cluster_name={{cluster}}) AND (job = 'kubelet')) SINCE 60 MINUTES AGO UNTIL NOW FACET cluster_name LIMIT max TIMESERIES 30000"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "yAxisLeft": {
+              "zero": false
+            }
+          }
+        },
+        {
+          "title": "Goroutines",
+          "layout": {
+            "column": 9,
+            "row": 37,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT latest(go_goroutines) FROM Metric WHERE ((cluster_name={{cluster}}) AND (job = 'kubelet')) SINCE 60 MINUTES AGO UNTIL NOW FACET cluster_name LIMIT max TIMESERIES 30000"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "yAxisLeft": {
+              "zero": false
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "variables": [
+    {
+      "name": "cluster",
+      "items": null,
+      "defaultValues": [],
+      "nrqlQuery": {
+        "accountIds": [
+          11732014
```

Comment:
last one here

Reply:
done, thank you:)

Reply:
@csongnr I think we're just missing the santize on this specific dashboard! The other dashboard looks good though

Reply:
oh i see, this was a different dashboard :) just ran it now.